app/src/main/java/com/canvearth/canvearth/client/PixelManager.java

```diff
@@ -0,0 +1,58 @@
+package com.canvearth.canvearth.client;
```

Comment:
Since we have PixelDataManager class, I think this class name can be confusing. How about to rename PixelDrawManager (or something else) ?

Reply:
Changed to GridManager